source/pci/pci.go

```diff
@@ -146,3 +154,25 @@ func detectPci() (map[string][]pciDeviceInfo, error) {
 
 	return devInfo, nil
 }
+
+// Check if the system has a remote DMA capable device.
+// Use PCI vendor ID for now.
+func hasRdmaCapableDevice(devs map[string][]pciDeviceInfo) bool {
+	for _, classDevs := range devs {
+		for _, dev := range classDevs {
+			if contains(rdmaCapablePCIVendorIds, dev["vendor"]) {
+				return true
+			}
```

Comment:
For other vendors, we might need to filter by more PCI attributes, e.g. Intel has a variety of different cards.

Reply:
I can filter by  both vendor id and device ID as they uniquely identify a PCI device.
allowing "wild card"/regex for the device ID.